mappings/net/minecraft/world/biome/source/util/MultiNoiseUtil.mapping

```diff
@@ -0,0 +1,213 @@
+			COMMENT Creates a new {@link MultiNoiseUtil.ParameterRange} that combines the parameters.
+			COMMENT
+			COMMENT @return the created parameter range
+			ARG 0 min
+				COMMENT the range that will be used for the created range's minimum value
+			ARG 1 max
+				COMMENT the range that will be used for the created range's maximum value
+	CLASS C_nakhdssm NoiseHypercube
+		COMMENT Represents a hypercube in a multi-dimensional cartesian plane.
+		COMMENT The multi-noise biome source picks the closest noise hypercube
+		COMMENT from its selected point and chooses the biome associated to it.
+		FIELD f_aeomxhyn offset F
+			COMMENT This value works differently from the other parameters, in that it is
+			COMMENT always {@code 0} during biome generation and does not use noise.
+			COMMENT
+			COMMENT <p>This means that setting it to a non-null number will make the biome smaller.
+			COMMENT The farther {@code offset} is from {@code 0}, the smaller the biome will be.
+			COMMENT For this, it does not matter whether {@code offset} is positive or negative.
+		FIELD f_bqnbodjl CODEC Lcom/mojang/serialization/Codec;
+		FIELD f_ipiwcjsb erosion Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+		FIELD f_izgxpchr humidity Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+		FIELD f_jjtllnxu continentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+		FIELD f_makaqamj depth Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+		FIELD f_mgirptbw weirdness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+		FIELD f_zsiprdpt temperature Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+		METHOD <init> (Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;F)V
+			ARG 1 temperature
+			ARG 2 humidity
+			ARG 3 continentalness
+			ARG 4 erosion
+			ARG 5 depth
+			ARG 6 weirdness
+			ARG 7 offset
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 other
+		METHOD m_auugwbdu getSquaredDistance (Lnet/minecraft/unmapped/C_ygdawqho$C_spkivtyd;)F
+			COMMENT Calculates the distance from this noise point to another one. The
+			COMMENT distance is a squared distance in a multi-dimensional cartesian plane
+			COMMENT from a mathematical point of view, with a special parameter that
+			COMMENT reduces the calculated distance.
+			COMMENT
+			COMMENT <p>For most fields except weight, smaller difference between
+			COMMENT two points' fields will lead to smaller distance. For weight,
+			COMMENT larger differences lead to smaller distance.
+			COMMENT
+			COMMENT <p>This distance is used by the mixed-noise biome layer source. The
+			COMMENT layer source calculates an arbitrary noise point, and selects the
+			COMMENT biome that offers a closest point to its arbitrary point.
```

Comment:
I'm not actually sure how this got into Yarn, but there a few issues with this.

- the special parameter is called `offset`, not `weight`
- it says "difference between two points' fields", but the points aren't compared with each other, only with the actual noise values at the end
- If offset is set to a non-null value, the distance actually gets larger, not smaller.
- It's called "multi-noise biome source", not "mixed-noise biome layer source"